pkg/utils/resultconfigmap.go

```diff
@@ -10,12 +12,25 @@ import (
 	"github.com/openshift/compliance-operator/pkg/controller/common"
 )
 
-func encodetoBase64(str []byte) string {
-	return base64.StdEncoding.EncodeToString(str)
+func encodetoBase64(src io.Reader) string {
+	pr, pw := io.Pipe()
+	enc := base64.NewEncoder(base64.StdEncoding, pw)
+	go func() {
+		_, err := io.Copy(enc, src)
+		enc.Close()
+
+		if err != nil {
+			pw.CloseWithError(err)
+		} else {
+			pw.Close()
+		}
+	}()
+	out, _ := ioutil.ReadAll(pr)
```

Comment:
@jhrozek instead of reading the whole content into memory, then encoding it (which will read it again). This sets up the encoder to pass this directly to a pipe. And so we encode, then just read from the pipe. The go routine is required for working with the pipe; else it blocks.